meson.build

```diff
@@ -2678,11 +2678,11 @@ foreach test: tests
       'env': env,
       'args': [
         testwrap.path(),
-        t['sd'],
-        meson.build_root(),
-        t['bd'],
-        t['name'],
-        'pg_regress',
+        '--srcdir', t['sd'],
+        '--basedir', meson.build_root(),
+        '--builddir', t['bd'],
+        '--testgroup', t['name'],
+        '--test_name', 'pg_regress',
```

Comment:
Looks a bit odd to have 'srcdir' etc (i.e. words concatenated together without separator), but then 'test_name' (i.e. concatenated with `_` as separator).